### PR TITLE
Day.js initialization for mantine

### PR DESCRIPTION
**Pull request by ranquild**
How to verify:
- The app starts in different locales without exceptions

### CODE REVIEW

File: frontend/src/metabase/lib/i18n.js [hunk 1]
```diff
@@ -26,8 +27,8 @@ export async function loadLocalization(locale) {
   setLocalization(translationsObject);
 }
 
-// Tell Moment.js to use the value of the start-of-week Setting for its current locale
-export function updateMomentStartOfWeek() {
+// Tell moment.js and day.js to use the value of the start-of-week Setting for its current locale
+export function updateMomentAndDayjsStartOfWeek() {
```
Comment (uladzimirdev) [inline frontend/src/metabase/lib/i18n.js:31]: nit: I'd have two separate functions: `updateMoment*` and `updateDayjs*` and then a common `updateStartOfWeek` which contains both those functions. So upcoming removal of moment will be less painful

**Comment by ranquild** [comment]
@uladzimirdev wdyt about a file with dayjs plugins that gets imported into the app and jest? Does it seem like a way forward?

**Comment by uladzimirdev** [comment]
> @uladzimirdev wdyt about a file with dayjs plugins that gets imported into the app and jest? Does it seem like a way forward?

sounds good to me

File: frontend/src/metabase/lib/i18n.js [hunk 2]
```diff
@@ -62,35 +57,66 @@ function setLanguage(translationsObject) {
...
     case "zh":
     case "zh-Hans":
       return "zh-cn";
     default:
-      return locale.toLowerCase();
+      return language.toLowerCase();
+  }
+}
+
+function getStartOfWeekDay() {
```
Comment (ranquild) [inline frontend/src/metabase/lib/i18n.js:106]: Moved without changes

**Review by uladzimirdev — APPROVED** [review]
LGTM

**Pull request merged**